react/components/atoms/button/button.js

```diff
@@ -83,15 +114,22 @@ export class Button extends mix(PureComponent).with(IdentifiableMixin) {
                 disabled={this.props.disabled}
                 onPress={this.props.onPress}
             >
-                <LinearGradient
-                    style={this._linearGradientStyle()}
-                    angle={this.props.gradientAngle}
-                    colors={this.props.gradientColors}
-                    locations={this.props.gradientLocations}
-                    useAngle={true}
-                >
-                    {this.props.loading ? this._renderLoading() : this._renderNormal()}
-                </LinearGradient>
+                {!this.props.backgroundColor && (
+                    <LinearGradient
+                        style={this._linearGradientStyle()}
+                        angle={this.props.gradientAngle}
+                        colors={this.props.gradientColors}
+                        locations={this.props.gradientLocations}
+                        useAngle={true}
+                    >
+                        {this.props.loading ? this._renderLoading() : this._renderNormal()}

```

Comment:
This condition is repeated
I think it's because you just want to change the background color. If so, I would just move this block of code to a method to better encapsulate the logic and simplify the repeated code